argilla/docs/tutorials/text_classification.ipynb

```diff
@@ -0,0 +1,529 @@
+{
```

Comment:
<p>Voilà! We have added the suggestions to the dataset, and they will appear in the UI marked with a ✨. </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5055/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>